src/frontend/Elaborator.ml

```diff
@@ -202,6 +205,21 @@ and chk_tm : CS.con -> T.Chk.tac =
         | D.TpDim, _, _ -> RM.ret @@ R.Dim.literal n
```

Comment:
To explain my other comment, this is the line where you can add another case, and call something like `R.DDim.literal`.